bindgen/features.rs

```diff
@@ -170,6 +170,42 @@ pub const LATEST_STABLE_RUST: RustTarget = {
     }
 };
 
+/// Earliest stable release of Rust that is supported by bindgen
+pub const EARLIEST_STABLE_RUST: RustTarget = {
+    // FIXME: replace all this code by
+    // ```
+    // RustTarget::stable_releases()
+    //     .into_iter()
+    //     .min_by_key(|(_, m)| m)
+    //     .map(|(t, _)| t)
+    //     .unwrap_or(LATEST_STABLE_RUST)
+    // ```
+    // once those operations can be used in constants.
+    let targets = RustTarget::stable_releases();
+
+    let mut i = 0;
+    let mut earliest_target = None;
+    let Some(mut earliest_minor) = LATEST_STABLE_RUST.minor() else {
```

Comment:
Maybe `LATEST_STABLE_RUST.minor().expect("No minor version?")`? I guess it has to be this way for `const`-ness?

Reply:
yeah it is for const-ness